mne/event.py

```diff
@@ -745,14 +745,20 @@ def make_fixed_length_events(raw, id, start=0, stop=None, duration=1.):
         of the recording.
     duration: float
         The duration to separate events by.
+    first_samp: bool
+        If True (default), times will be adjusted for first_samp. The default
+        behavior is desirable if times are arbitrary, as in resting data. Use 
```

Comment:
```
This behavior is not desirable if times were generated by :func:mne.find_events.
```

This is the part that I don't think represents the issue globally enough. This is true for your use case, I think, because you plan to take the output and add it to some existing event that has first_samp in it already. But for most users who might use `find_events` to do some event-based stuff, and `make_fixed_length_events` to do some other things (without combining the outputs of the two), this statement is incorrect. How about:
```
This behavior is not desirable if the returned events will be combined with event times
that already have ``raw.first_samp`` added to them, e.g. event times that come from
:func:`mne.find_events`.
```
I don't think the main tagline needs to change -- changing the docstring for the argument should be good enough.